src/addrdb.cpp

```diff
@@ -220,7 +220,6 @@ std::vector<CAddress> ReadAnchors(const fs::path& anchors_db_path)
         anchors.clear();
     }
 
-    fs::remove(anchors_db_path);
```

Comment:
It look like this commit isn't standalone and should be combined so as to maintain the file deletion functionality.

Reply:
Done, thank you!